Extensions/combined/_locales/fr/messages.json

```
@@ -0,0 +1,35 @@
+{
+    "extensionName": {
+        "message": "Retour YouTube Dislike"
+    },
+    "extensionDesc": {
+        "message": "Rend la capacité de voir les aversions"
+    },
+    "textDeveloper": {
+        "message": "par Dmitrii Selivanov & Communauté"
+    },
+    "linkWebsite": {
+        "message": "Site web"
+    },
+    "linkFAQ": {
+        "message": "FAQs"
```

```suggestion
        "message": "Questions fréquentes"
```
FAQ could be used instead too, though it usually means Q&A in french, in my experience, refering to a forum where people can ask whichever questions they want…